src/models/daily-dropoffs-report-data.ts

```diff
@@ -158,6 +158,8 @@ function loadData(
   env: string,
   fetch = window.fetch
 ): Promise<DailyDropoffsRow[]> {
+  // Set the "finish" to the ending of the day
+  finish.setUTCHours(23, 59, 59, 999);
```

Comment:
1. looks like this mutates the parameter to the function? I think that's a pattern we should try to avoid, can we make a copy of the param if we're going to mutate it?

2. Is there a way we can use `utcDay.ceil()` or some similar helper function to advance this?

Reply:
I've used the timeDay.Ceil() function to accomplish Item 2. Think it's still necessary to make a copy of the param while utilizing the .ceil() function?

Reply:
yup! AFAIK `.ceil` should make a new instance